apache/uima-uimacpp
# Attempt to separate python annotator from Python extension

**JIRA Ticket:** https://issues.apache.org/jira/browse/UIMA-XXXX

**What's in the PR**
* A new directory  calles changes in the python directory which contains new files python_annotator.cpp, python_modules.cpp and a Makefile which builds them. These are files extracted from splitting pythonnator.cpp.

**How to test manually**
* Building the DLLs using the make commands then checking if the DLLs (_python_annotator.so and _python_modules.so) are generated in your directory, and finally using make clean to verify their clean up.

**Automatic testing**
* [ ] PR adds/updates unit tests -no

**Documentation**
* [ ] PR adds/updates documentation -no

**Organizational**
- [ ] PR includes new dependencies.   -no
      <sub><sup>Only dependencies under [approved licenses](http://www.apache.org/legal/resolved.html#category-a) are allowed. LICENSE and NOTICE files in the respective modules where dependencies have been added as well as in the project root have been updated.</sup></sub>
